### DISCUSSION TITLE
Is possibile to open the commit editor on 'git commit' command?

### DISCUSSION THREAD

**Discussion by nicosh** [bendera/vscode-commit-message-editor#49] [Q&A]
As the title says, if i use VScode as text editor for commits, when i prompt on my shell `git commit` (without -m) it will open a new VScode  tab, is possibile to open the editor instead that default tab?
Thanks

**Comment by bendera** [comment]
As far as I know, it is not possible. But honestly, if you can configure and use the VSCode this way, it's a better idea to use the built-in editor instead of a textarea in the webview tab (which is used in the extension).

(Sorry for late response, I haven't been notified about this question.)

**Comment by bendera** [comment]
UPDATE: Rebase editor of GitLens works the same way so it can be solved somehow.
